styleguide/derek/pattern-components/challenges-river/challenges-river.ejs

```diff
@@ -0,0 +1,69 @@
+    },
+    {
+      shape: '<?xml version="1.0" encoding="utf-8"?><svg class="rackshape" version="1.1" id="Layer_1" x="0px" y="0px" width="201px" height="201px" viewBox="0 0 132 132" style="enable-background:new 0 0 201 201;" xmlns="http://www.w3.org/2000/svg"><style type="text/css">.st0{fill:none;stroke:#EB0000;stroke-miterlimit:10;}</style><g transform="matrix(1, 0, 0, 1, -34.5, -34.5)"><circle class="st0" cx="148.5" cy="52.5" r="16"/><circle class="st0" cx="52.5" cy="52.5" r="16"/><circle class="st0" cx="100.5" cy="52.5" r="16"/><circle class="st0" cx="148.5" cy="100.5" r="16"/><circle class="st0" cx="52.5" cy="100.5" r="16"/><circle class="st0" cx="100.5" cy="100.5" r="16"/><circle class="st0" cx="148.5" cy="148.5" r="16"/><circle class="st0" cx="52.5" cy="148.5" r="16"/><circle class="st0" cx="100.5" cy="148.5" r="16"/></g></svg>',
+      title: 'Regulate and comply',
+      copy: 'Address your industry’s regulations in a compliant, custom environment.'
+    },
+    {
+      shape: '<?xml version="1.0" encoding="utf-8"?><svg class="rackshape" version="1.1" id="Layer_1" x="0px" y="0px" width="201px" height="201px" viewBox="0 0 68 132" style="enable-background:new 0 0 201 201;" xmlns="http://www.w3.org/2000/svg"><style type="text/css">.st0{fill:none;stroke:#EB0000;stroke-miterlimit:10;}</style><g transform="matrix(1, 0, 0, 1, -66.5, -34.5)"><circle class="st0" cx="100.5" cy="132.5" r="32"/><circle class="st0" cx="100.5" cy="68.5" r="32"/><circle class="st0" cx="100.5" cy="100.5" r="32"/></g></svg>',
+      title: 'Uptime and performance',
+      copy: 'Avoid downtime with a customized infrastructure built for speed and stability.'
+    },
+    {
+      shape: '<?xml version="1.0" encoding="utf-8"?><svg class="rackshape" version="1.1" id="Layer_1" x="0px" y="0px" width="201px" height="201px" viewBox="0 0 132 124" style="enable-background:new 0 0 201 201;" xmlns="http://www.w3.org/2000/svg"><style type="text/css">.st0{fill:none;stroke:#EB0000;stroke-miterlimit:10;}</style><g transform="matrix(1, 0, 0, 1, -34.5, -38.5)"><circle class="st0" cx="128.5" cy="124.5" r="36"/><circle class="st0" cx="72.5" cy="124.5" r="36"/><circle class="st0" cx="100.5" cy="76.5" r="36"/></g></svg>',
+      title: 'Consolidate data centers',
+      copy: 'Replace your aging hardware infrastructure with powerful modern architecture.'
+    },
+    {
+      shape: '<?xml version="1.0" encoding="utf-8"?><svg class="rackshape" version="1.1" id="Layer_1" x="0px" y="0px" width="201px" height="201px" viewBox="0 0 132 132" style="enable-background:new 0 0 201 201;" xmlns="http://www.w3.org/2000/svg"><style type="text/css">.st0{fill:none;stroke:#EB0000;stroke-miterlimit:10;}</style><g transform="matrix(1, 0, 0, 1, -34.5, -34.5)"><circle class="st0" cx="100.5" cy="68.5" r="32"/><circle class="st0" cx="100.5" cy="132.5" r="32"/><circle class="st0" cx="132.5" cy="100.5" r="32"/><circle class="st0" cx="68.5" cy="100.5" r="32"/></g></svg>',
```

Comment:
Can we pull out the `<?xml version="1.0" encoding="utf-8"?>` tags here?   Turns out we don't need them in with embedded svgs.